gitlab-ci.yml

```diff
@@ -0,0 +1,42 @@
+cache:
+  key:
+    files:
+      # Cache npm packages using lock file
+      - package-lock.json
+  paths:
+    - .npm/
+
+before_script:
+    - npm ci --cache .npm --legacy-peer-deps --loglevel error
+
+stages:
+  - lint
+  - test
+
+reviewdog:
+  image: node:16.16
+  stage: lint
+  script:
+    - mkdir -p ~/bin/ && export PATH="~/bin/:$PATH"
+    - curl -sfL https://raw.githubusercontent.com/reviewdog/reviewdog/master/install.sh| sh -s -- -b ~/bin
+    - export REVIEWDOG_GITLAB_API_TOKEN=$CODE_REVIEW_SHIFU_KEY
```

Comment:
Let's add comment here.
$CODE_REVIEW_SHIFU_KEY is the variable need to be added over gitlab CI variable.